README.md

```diff
@@ -1,5 +1,13 @@
 # `OpenEdu` Methodology Repository
 
+[![copying.md](https://img.shields.io/static/v1?label=license&message=CCBY-SA4.0&color=%23385177)](https://github.com/open-education-hub/methodology/blob/master/COPYING.md)
+[![copying.md](https://img.shields.io/static/v1?label=license&message=BSD-3-clause&color=%23385177)](https://github.com/open-education-hub/methodology/blob/master/COPYING.md)
```

Comment:
Shouldn't you name these 2 "license"? Or is there a reason why they muat be named "copying.md"?

Reply:
The reason is because we host both licenses in the same file. That's why I prefer `COPYING.md`. Both are handled by GitHub.